docker/dockerfiles/artifacts/artifact.Dockerfile

```diff
@@ -49,7 +49,7 @@ ADD 'https://dtdg.co/latest-java-tracer' /datadog/dd-java-agent.jar
 
 # Get ddprof for BE profiling
 RUN imagearch=$(arch | sed 's/aarch64/arm64/; s/x86_64/amd64/') \
-    && wget --no-check-certificate "https://github.com/DataDog/ddprof/releases/download/v0.15.3/ddprof-0.15.3-${imagearch}-linux.tar.xz" -O ddprof-linux.tar.xz \
+    && wget --no-check-certificate "https://github.com/DataDog/ddprof/releases/latest/download/ddprof-${imagearch}-linux.tar.xz" -O ddprof-linux.tar.xz \
```

Comment:
is it possible to have an ARG defined for the download link so we don't have to update the dockerfile whenever there is a need to upgrade the ddprof binary?

Reply:
also using `latest` might not be a good idea, it will cause the same dockerfile, even the same code commit, produces different docker image (different version of ddprof binary) in different time. Not a good practice for production env.

Reply:
It's suggested in [doc](https://docs.datadoghq.com/profiler/enabling/) to use the latest version, we follow it here for both java and c++ profile. It's always easier to maintain instead of pinning the version, and not necessary as a basic tool. Nevertheless, I don't have a strong opinion on it, would leave the question to you either use `latest` or other alternatives.